CONTRIBUTING.md

```diff
@@ -73,6 +73,10 @@ Continuous integration builds the project, runs the tests, and runs multiple
 types of static analysis.
 
 1. Note: Currently, to run the full suite of tests, you'll need to be running a docker daemon.
+The tests that require docker are disabled by default. If you wish to run them,
+you can enable the docker tests by setting a gradle property of
+``"enable.docker.tests"`` to true. See the gradle.properties file in the root of the project
```

Comment:
The property name looks a bit odd to me, I'd actually expect it to be the other way round since the periods usually resemble some kind of hierarchy but that's just a nitpick 🙂

Reply:
yeah, I just made something up. I don't feel strongly about it.